internal/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -1,74 +1,15 @@
-
-Producing blocks in a proof-of-stake (PoS) Blockchain usually requires a staker to own vast amounts of staked tokens. Often only a few selected or fixed Validators are granted the ability to be block producers. These entities are pre-defined by the protocol's creators (a form of centralization) or emerge as validators holding the most significant stake of tokens. PoS blockchains do not require enormous amounts of energy wasted for PoW, but therefore only allow the wealthiest participants to write to the ledger and produce new blocks.
-
-## Consensus in the Tangle
-
-Sending messages requires no fees in the Tangle because the network has no miners or stakers. In the Tangle, PoW is not used to secure the network. Instead, PoW is used only to discourage spam messages. All IOTA nodes validate messages and use different functions alongside messages in their confirmation to reach consensus. Currently, messages will only be considered valid if they reference a milestone. These milestones are issued by a special network node – the Coordinator.
-
-## The Coordinator
-
-The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. This topic describes how nodes use milestones to determine which messages are confirmed.
-
-The Coordinator is a temporary solution. IOTA will remove the Coordinator in the upcoming IOTA 2.0 network. The following text describes how the Coordinator is currently deployed within IOTA.
-
-Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
-
-To allow the nodes to recognize valid milestones, all nodes that participate in the same IOTA network are configured with the Ed25519 signatures of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted Coordinator did sign them.
-
-To ensure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This way ensures that nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
-
-![coordinator](/img/learn/milestones.gif)
+The IOTA protocol is still in research. It has two public networks: the IOTA mainnet is the stable network that manages your IOTA tokens, and the Shimmer testnet is the staging network for the latest and greatest protocol updates. Once the changes are well tested on Shimmer, they come to the mainnet. The next big update is [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/). The future update that concludes decentralization effort is called [Coordicide](roadmap-to-decentralization.md).
```

Comment:
could you add links to IOTA mainnet and Shimmer ?

Reply:
I've added for the ../future/shimmer.md, but I'm not sure what to reference for the mainnet.